server/index.js

```diff
@@ -90,6 +90,21 @@ app.get("/v2/query", async (req, res) => {
   res.send({ location, results });
 });
 
+app.get("/v2/queryByStoreId", async (req, res) => {
+  const { query } = req;
+  let params = {
+    storeId: query.storeId,
+    radius: query.radius,
+    page: query.page
+  }
+  let results = await stores.findNearbyStoreByStoreId(params)
+  let location = {
+    lat: results[0].Latitude,
+    lng: results[0].Longitude
```

Comment:
i believe this is assuming that the first result returned will be the location of the store, which is ok for now, but adds a complexity to the `findNearbyStores` method - it would always need to return a sorted list. it would be nicer if we could not rely on that assumption